### PR TITLE
Bug/#215 employee equipment  when adding two popups show/closed

### LINKED ISSUE #215: [Bug] employee equipment  when adding two popups show

### PR DESCRIPTION
**Pull request by JavaLavaMT**
closed #215

### CODE REVIEW

File: app/templates/account/employee/onboard/equipment.hbs [hunk 1]
```diff
@@ -51,13 +51,6 @@
         <label for="asset-identifier">Asset Identifier</label>
         {{input type="text" value=pendingAssetItem.identifier id="asset-identifier" placeholder="Unique identifier (VIN, Stock #, RFID)"}}
       </div>
-
-      {{#if pendingAssetItem.asset.attributes}}
-        <div class="ui divider"></div>
-        {{#each pendingAssetItem.asset.attributes as |attribute|}}
-          {{input/custom-field object=pendingAssetItem.customFields attribute=attribute}}
-        {{/each}}
-      {{/if}}
```
Comment (James1x0) [inline app/templates/account/employee/onboard/equipment.hbs]: We definitely still need this block of code. It's responsible for handling attributes set on each asset item. Add a test attribute to your asset category and attempt to fill this when creating an action item. You won't see any attributes with this missing.

**Comment by James1x0** [comment]
@petersonbrock Is this even an issue anymore?

**Comment by petersonbrock** [comment]
@James1x0 I'm not sure I was involved with this issue, so I don't know for sure

**Comment by James1x0** [comment]
@petersonbrock https://github.com/associatedemployers/granite/pull/238

**Comment by James1x0** [comment]
@petersonbrock 👍

**Pull request closed without merging**